ogre2/src/Ogre2Heightmap.cc

```diff
@@ -211,9 +225,10 @@ void Ogre2Heightmap::Init()
 
   const math::Vector3d size = this->descriptor.Size();
 
+  // The position's Y sign ends up flipped
   math::Vector3d center(
       this->descriptor.Position().X(),
-      this->descriptor.Position().Y(),
+      -this->descriptor.Position().Y(),
```

Comment:
@darksylinc , I see you added Ogre 2 heightmaps in https://github.com/ignitionrobotics/ign-rendering/pull/386. I'd appreciate some feedback here, not sure I'm misunderstanding anything.

Reply:
I wasn't 100% sure of the alignment(s)/scale(s) either, as some of the code and comments were conflicting and I didn't have enough examples to work with.

When it comes to aligning stuff, sadly it always ends up randomly swapping components and signs until it works.

Reply:
Gotcha 👍🏽 I think this is the right move then.